src/components/landing/GroupLogin.jsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import LandingCard from './LandingCard.jsx';
+import TextInput from '../TextInput.jsx';
+import Button from '../Button.jsx';
+import '../../scss/landing.scss';
+
+/// callBack should be passed from landing page. Used to re-render the landing page
+/// with the default landing page card, which should be the login card.
+
+function GroupLogin({ onLoginClick }) {
+  let input;
+
+  function submitCode(event) {
+    event.preventDefault();
+    // TODO: Make login request
+  }
+
+  return (
+    <LandingCard title="No account? No problem.">
+      <form onSubmit={submitCode}>
```

Comment:
The `onSubmit` can probably be removed as it's now embedded onto the button

Reply:
Got it

Reply:
Just from the form or should submitCode be removed entirely?

Reply:
Just from the form element